test/unit/readywait.js

```diff
@@ -0,0 +1,12 @@
+var start = new Date();
```

Comment:
What's the purpose of this file? It runs the same test as one in test/unit/ready and it's not used anywhere, this looks like dead code.